sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsyncBenchmark.java

```diff
@@ -56,15 +60,17 @@
     Timer latency;
 
     AsyncBenchmark(Configuration cfg) {
-        client = new AsyncDocumentClient.Builder()
-            .withServiceEndpoint(cfg.getServiceEndpoint())
-            .withMasterKeyOrResourceToken(cfg.getMasterKey())
-            .withConnectionPolicy(cfg.getConnectionPolicy())
-            .withConsistencyLevel(cfg.getConsistencyLevel())
-            .build();
+        v4Client = new CosmosClientBuilder()
+            .setEndpoint(cfg.getServiceEndpoint())
+            .setKey(cfg.getMasterKey())
+            .setConnectionPolicy(cfg.getConnectionPolicy())
+            .setConsistencyLevel(cfg.getConsistencyLevel())
+            .buildAsyncClient();
 
         logger = LoggerFactory.getLogger(this.getClass());
 
+        client =  BridgeInternalBenchmark.getInternalDocumentClient(v4Client);
```

Comment:
If we want to use CosmosClient, then why bother getting internal `AsyncDocumentClient` ?
Shouldn't we change all the operations to use `v4Client` and remove `AsyncDocumentClient` completely from benchmark. I thought that is the goal for this PR.

Reply:
there are many different benchmarks ReadMyWrites, Write, etc. The goal of this PR is to just make the change for the `AsyncReadBenchmark`. If we change client vs v4Client this will become a cascading change to all other benchmarks which we don't want. as the scope of this PR is only `AsyncReadBenchmark` and `AsyncReadBenchmark` internally now uses v4Client for reads.

Reply:
Makes sense.